presto-benchto-benchmarks/README.md

```diff
@@ -67,6 +68,8 @@ macros:
 With the scene set up as in the previous section, the benchmark can be run with:
 ```bash
 ./mvnw clean package -pl presto-benchto-benchmarks
-java -Xmx1g -jar presto-benchto-benchmarks/target/presto-benchto-benchmarks-*-executable.jar \
+java -Xmx1g -jar path/to/benchto-driver.jar \
+    --sql presto-benchto-benchmarks/src/main/resources/sql \
+    --benchmarks presto-benchto-benchmarks/src/main/resources/benchmarks \
```

Comment:
it would be nice if this would be a complete script that user can just copy-paste and have something running without changing anything